open_samus_returns_rando/patcher_editor.py

```diff
@@ -56,3 +56,10 @@ def copy_actor(self, scenario: str, coords, templateActor: Container, newName: s
         newActor.z = coords[2] + offset[2]
 
         return newActor
+    
+    def remove_entity(self, reference: dict):
```

Comment:
The actor should also be removed from the actor group / entity group or whatever you want to call it.
Basically the inverse of where we add them like for the spawn points: https://github.com/randovania/open-samus-returns-rando/blob/b90cf9f1970132d8e1fe1e7cb06c6be6591e7318/open_samus_returns_rando/samus_returns_patcher.py#L89-L91

Reply:
My new PR should help with that. Could use `remove_actor_from_all_groups`